src/react-guides/Guides.tsx

```diff
@@ -495,11 +495,14 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
         if (!datas.fromRuler || !this._isFirstMove) {
             if (displayDragPos) {
                 const displayPos = type === 'horizontal'
-                    ? [offsetX, nextPos]
-                    : [nextPos, offsetY];
+                ? [offsetX, nextPos]
+                : [nextPos, offsetY];
+                const transform = type === 'horizontal' 
+                    ? `translate(${displayPos.map((v, i) => !i ? '-6px' : `${v - 13}px` ).join(', ')}) rotate(-90deg)` 
+                    : `translate(${displayPos.map((v, i) => !i ? `${v + 16}px` : '9px' ).join(', ')})`;
```

Comment:
I think will be good if you can move these calculations to the functions, so it will be easier to understand a code